corehq/apps/geospatial/static/geospatial/js/case_management.js

```diff
@@ -526,12 +526,12 @@ hqDefine("geospatial/js/case_management", [
                     'danger',
                 );
             }
-        } else if (xhr.responseJSON.aaData.length && mapModel.mapInstance) {
+        } else if (xhr.responseJSON.data.length && mapModel.mapInstance) {
```

Comment:
Where did `aaData` get renamed to `data`? Or did something different happen here?